File: app/src/main/java/io/blacklagoonapps/myquotes/settings/SettingsFragment.java
package io.blacklagoonapps.myquotes.settings;

import android.os.Bundle;
import android.support.v7.preference.PreferenceFragmentCompat;

import io.blacklagoonapps.myquotes.R;


public class SettingsFragment extends PreferenceFragmentCompat{

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String s) {
        addPreferencesFromResource(R.xml.preferences);
    }
}

File: app/src/main/java/io/blacklagoonapps/myquotes/settings/SettingsFragment.kt
package io.blacklagoonapps.myquotes.settings

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat

import io.blacklagoonapps.myquotes.R

class SettingsFragment : PreferenceFragmentCompat() {

    override fun onCreatePreferences(savedInstanceState: Bundle, s: String) {
        addPreferencesFromResource(R.xml.preferences)
    }
}
